### ISSUE TITLE
Loading Cormas on non-Moose image raises dependency exception

### ISSUE THREAD

**Issue by hernanmd** [cormas/cormas#25]
When loading Cormas in Pharo 6.1, an exception is raised with the following message:

```
This package depends on the following classes:
  RTView
You must resolve these dependencies before you will be able to load these definitions: 
  CMPharoGridSimulation
  CMPharoGridSimulation>>#on:with:
  CMPharoGridSimulation>>#runAndVisualize

Select Proceed to continue, or close this window to cancel the operation.
```

Fixing this warning by resolving the dependency will enable to load Cormas on standard Pharo image or even minimal images.

**Comment by SergeStinckwich** [comment]
The only dependency of CORMAS is with ROASSAL. So it should be possible to build a configuration in order to load CORMAS in a Pharo 6.1 image.

**Comment by hernanmd** [comment]
Already fixed in releases >= 0.29
